dynamodb/README.md

```diff
@@ -0,0 +1,134 @@
+# HarbourBridge: Turnkey DynamoDB-to-Spanner Evaluation
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL or MySQL or DynamoDB database. This
+README provides details of the tool's DynamoDB capabilities. For general
+HarbourBridge information see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-turnkey-spanner-evaluation).
+
+## Example DynamoDB Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+HarbourBridge can be run directly on a DynamoDB database (via the aws go sdk
+package).
+
+Before running HarbourBridge, make sure that you have
+[set up your AWS credentials/region](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+correctly.
+
+To use the tool directly on a DynamoDB database (it will migrate all tables),
+run
+
+```sh
+harbourbridge -driver=dynamodb
+```
+
+Due the schemaless nature of DynamoDB, the tool infers the schema based on a
+certain amount of sampled data, by default, 100000 rows. If a table has more
+rows than the default value, we only use 100000 rows for estimating the schema.
+You can change this value, run
+
+```sh
+harbourbridge -driver=dynamodb -schema-sample-size=1000000
```

Comment:
It'd hard to see the difference between 100000 and 1000000.
Suggest changing 1000000 to 500000 to make this clearer.